src/sunrise/core.cljs

```diff
@@ -1,13 +1,8 @@
 (ns sunrise.core
   (:require [reagent.core :as reagent]
-            [sunrise.component :as component]))
+            [sunrise.component :as component]
+            [sunrise.state]))
 
-(defn get-window-dimensions []
-  (let [width (.-innerWidth js/window)
-        height (.-innerHeight js/window)]
-    {:width width :height height}))
+(.addEventListener js/window "resize" sunrise.state/update!)
```

Comment:
I'm curious, why add the event listener in a different namespace than `get-window-dimensions`. Here are a few reasons why I don't love it:

1. The event listener and the dimensions function both are concerned with `js/window`
2. I think `update!` might be too much of an abstraction, as I imagine we will have other state that will change.

Do you have any thoughts on those two criticisms?